roles/bigboot/tasks/main.yaml

```diff
@@ -21,17 +25,6 @@
     boot_device_name: "{{ _boot_device.device | regex_replace('[0-9]*', '') }}"
```

Comment:
@ygalblum, this regex doesn't correctly identify the parent block device in all cases. Consider this example...
```
[root@steady-midge ~]# lsblk -sp /dev/nvme0n1p1
NAME           MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
/dev/nvme0n1p1 259:1    0  476M  0 part /boot
└─/dev/nvme0n1 259:0    0   30G  0 disk 
```
The following works for me... 
```yaml
- name: Identify the boot device parent
  ansible.builtin.command: "lsblk -spnlo name {{ _boot_device.device }}"
  changed_when: false
  register: _boot_device_deps

- name: Capture boot device details
  ansible.builtin.set_fact:
    boot_device_name: "{{ _boot_device_deps.stdout_lines | last }}"
    boot_device_original_size: "{{ _boot_device.size_total | int }}"
```